golang/cosmos/proto/agoric/swingset/swingset.proto

```diff
@@ -150,8 +150,8 @@ message Egress {
     ];
 }
 
-// The payload messages used by swingset state-sync 
-message ExtensionSnapshotterArtifactPayload {
+// The artifacts of swing-store snapshots
+message StoreSnapshotArtifact {
```

Comment:
This type was only used internally, so it felt ok renaming. the protobuf breakage test is complaining, but it seems it's not a required test?

Reply:
You can silence the protobuf breakage CI with the `proto:expect-breakage` label.

Reply:
Looks like the `expect-breakage` only influences mergify and not the test run itself.